Description: Respond with a 401 HTTP code when IMAP login fails.
 By responding with a 401 HTTP code this allows sshguard/fail2ban/etc to
 monitor the logs and block remote address accordingly.
 .
 This will still respond with the same login error template,
 so the user experience should be no different.
Author: Mike Abrahall <mike@cyber.com.au>
Origin: <upstream|backport|vendor|other>,<S-Del> <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: 2024-11-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: prayer/servers/prayer_login.c
===================================================================
--- prayer.orig/servers/prayer_login.c
+++ prayer/servers/prayer_login.c
@@ -222,7 +222,7 @@ prayer_login_error(struct prayer *prayer
     template_vals_string(tvals, "$value", value);
 
     template_expand("frontend_login_error", tvals, b);
-    response_html(request, 200);
+    response_html(request, 401);
 }
 
 /* ====================================================================== */
